### ISSUE TITLE
IA app name cookie write failed The copy of the installer app failed. (251)

### ISSUE THREAD

**Issue by roto31** [chris1111/HP-Probook-EliteBook-Package-Creator#32]
Hello,
I have tried multiple times to create a USB installer with the Create Install Media HP Laptop application after building it with the applications in the package from your repository on GitHub. I get the error "IA app name cookie write failed The copy of the installer app failed. (251)" each time I try to run it. I am running it from a real Mac with SIP disabled which is running 10.15.4. It has happened trying it with Catalina and Mojave. Do I need to be running it from Mojave or is it supposed to run on Catalina? 

You mention that Create install is using the crreateinstall media terminal command? Would you be able to make the Clover installer separate from the Create Install Media package so it can be run outside the process? 

Would I be able to use "create install media" on it's own and then run the standalone installed that is the first git hub repo that you have listed?

I have used your packages in the past for Hackintosh installs. All have worked great :) 

Thank you

**Comment by chris1111** [comment]
I just test  from my MacBook this issue its new from 10.15.4 and maybe apple Bug
I have to make more test 
thanks for the catch

**Comment by chris1111** [comment]
Hey my previous  test is with the SIP enable so I just Retest with SIP disable and I don't have any issue its work perfectly
SO your SIP is not disable

**Comment by roto31** [comment]
So I was able to run it successfully from my MacPro 5,1 that was running Catalina 10.15.4 with dosdude's patcher for Catalina. The only thing I did not do was run the Gatekeeper disable. All steps were all able to complete. The machines it failed on were a 2015 MacBook Pro and a 2016 MacBook Pro both running Catalina 10.15.4.

I was able to install Catalina using this USB drive. The issue I am now running into now after the post install, is when running in verbose mode it stops after the first stage of starting up and ends with "** In Memory Panic Stackshot Succeeded ** Bytes Traced 10800 ** Attempting system restart MACH Reboot 

The above is not an error message I have not saw before. Thoughts?
